testing/test_dilateGT.py

```diff
@@ -15,7 +15,7 @@
 
 cudnn.benchmark = True
 
-GPU_NUMBER = 5
+GPU_NUMBER = 1
```

Comment:
[here](https://github.com/neuropoly/ivado-medical-imaging/pull/132/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt#diff-64f10db8dac07e1cb7f592cb33a24b7cR21) GPU_NUMBER is set to 0. What is the reasoning behind setting those to 0 or 1?

Reply:
I changed it to 0. The tests were created with different GPU numbers, but I believe there was no particular reasoning for that.